charts/opentelemetry-collector/values.yaml

```diff
@@ -199,7 +199,7 @@ extraVolumeMounts: []
 
 # Configuration for ports
 ports:
-  otlp:
+  grpc-otlp:
```

Comment:
This should probably be called `otlp-grpc` in order to be consistent with `otlp-http`

Reply:
no, it must be `grpc-xxx` as prefix:https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/#explicit-protocol-selection

Reply:
if we do not use `grpc-otlp`, we can use `appProtocol`

Reply:
I would opt to use `appProtocol` here instead.

Reply:
This is actually a breaking change for the users. So I would keep the existing name. otherwise, the renaming should be done in a backward compatible way